fineract-provider/src/main/java/org/apache/fineract/infrastructure/event/external/service/serialization/serializer/loan/LoanBusinessEventSerializer.java

```diff
@@ -46,6 +46,7 @@ public <T> boolean canSerialize(BusinessEvent<T> event) {
     protected <T> ByteBufferSerializable toAvroDTO(BusinessEvent<T> rawEvent) {
         LoanBusinessEvent event = (LoanBusinessEvent) rawEvent;
         LoanAccountData data = service.retrieveOne(event.get().getId());
+        data = service.fetchRepaymentScheduleData(data);
```

Comment:
@josehernandezfintecheandomx do you think we can get this data with the retrieveOne call directly (so like extending the native query)?